packages/components/src/components/domain/event/eventEnrolmentStatus/useEventEnrolmentStatus.tsx

```diff
@@ -0,0 +1,30 @@
+import { EnrolmentStatusLabel } from '../../../../constants/event-constants';
+import { useEventTranslation } from '../../../../hooks';
+import type { EventFields } from '../../../../types';
+
+export function getEnrolmentStatus(event: EventFields): EnrolmentStatusLabel {
+  const now = new Date();
+  const { remainingAttendeeCapacity, enrolmentStartTime, enrolmentEndTime } =
+    event;
+
+  // TODO: Add EnrolmentStatusLabel.queueable when we can resolve it
+  if (remainingAttendeeCapacity === 0) {
+    return EnrolmentStatusLabel.full;
+  } else if (enrolmentStartTime && new Date(enrolmentStartTime) > now) {
+    return EnrolmentStatusLabel.enrolmentNotStartedYet;
+  } else if (enrolmentEndTime && new Date(enrolmentEndTime) < now) {
+    return EnrolmentStatusLabel.enrolmentEnded;
+  }
+  return EnrolmentStatusLabel.enrollable;
```

Comment:
Isn't `EnrolmentStatusLabel.enrollable` a combination of `enrollable` and `not_enrollable_at_all`? As the same value is returned also when e.g.:
 - remainingAttendeeCapacity is null/undefined and today is between enrolmentStartTime and enrolmentEndTime
 - remainingAttendeeCapacity is a non-zero number and both enrolmentStartTime and enrolmentEndTime are null/undefined

Examples of possible ways to make this clearer IMHO:
 - Renaming the enum value to make it clearer what it means
 - Just returning an empty string here and putting a comment about not showing anything for the enrollable case and the not enrollable at all case
 - Making another enum value for the not enrollable at all case and handling it in the logic and translating it also as empty

Reply:
added an option